### ISSUE TITLE
Graphql query question

### ISSUE THREAD

**Issue by ThisIsNotTheUserYouAreLookingFor** [LuatixHQ/opencti#8963]
## Prerequisites

- [x] I read the [Deployment and Setup](https://docs.opencti.io/latest/deployment/overview) section of the OpenCTI documentation as well as the [Troubleshooting](https://docs.opencti.io/latest/deployment/troubleshooting) page and didn't find anything relevant to my problem.
- [x] I went through old GitHub issues and couldn't find anything relevant
- [x] I googled the issue and didn't find anything relevant

## Description

I am trying to get all malware and intrusion sets linked to indicators that we have created (own TI) together with the date that it happened.
I have 90% of the graphql query already, but I seem to be missing either graphql knowledge, or a boolean to check against :)
This would be handy to make strategic decisions in the long run, as we can see who/what has been historically targetting us.
 
This is the query:

```
query Indicators {
    indicators(
        filters: {
            mode: and
            filters: [
                {key: "creator_id",operator: eq,values: ["your own UID"]}
                { key: "entity_type", operator: eq, values: ["Indicator"] }
                { key: "created_at", operator: gte, values: ["2024-10-01T00:00:00.000Z"] }
            ]
            filterGroups: []
        }
        first: 100
    ) {
        edges {
            node {
                name
                stixCoreRelationships(toTypes: ["Malware", "Intrusion-Set"]){
                    edges {
                        node {
                            toType
                            to {
                                ... on IntrusionSet {
                                    name
                                    aliases
                                }
                                ... on Malware {
                                    name
                                }
                            }
                        }
                    }
                }
                created_at
            }
        }
    }
}
```

This works, but  shows pulls all indicators, regardless of having no edges that match the stixCoreRelationships filter. What I would like is to skip the indicator if there are no edges. Anyone has an idea how to accomplish this? If you have a UI view to do this, feel free to let me know how stupid I am 😎 

## Environment

1. OS K8S
2. OpenCTI version: latest
3. OpenCTI client: graphql
4. Other environment details:

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. run the filter above in a playground with some indicators linked to malware or an intrusion set and a lot of indicators that are not

**Comment by Kedae** [comment]
Hi @ThisIsNotTheUserYouAreLookingFor,

What you are trying to do is not possible.
Let me rephrase just to be sure to understand correctly:
- You want to get all Intrusion sets & malwares linked to specific indicators
- You want those indicators to be filtered on the condition that they have such relationships

Is that right ?

In that case it's not doable, why should GraphQL filter a parent result on a partial child result.

**Comment by nino-filigran** [comment]
Closing this issue since there is no acitivty. Feel free to reopen if you need.
